app/src/main/java/org/fossasia/openevent/app/data/Repository.java

```diff
@@ -17,6 +17,7 @@
 /**
  * General Repository class. To be generified in future
  */
+@SuppressWarnings("MissingOverride")
```

Comment:
Add override rather than suppressing

Reply:
@iamareebjamal Replacing it with override shows this error: `'@Override' not applicable to type`

Reply:
OK